data/json/items/armor.json

```diff
@@ -2909,6 +2909,28 @@
     "flags": [ "WATER_FRIENDLY", "SUN_GLASSES" ]
   },
   {
+    "id": "eclipse_glasses",
+    "type": "ARMOR",
+    "name": "pair of eclipse glasses",
+    "name_plural": "pairs of eclipse glasses",
+    "description": "After the last eclipse before the Cataclysm stirred up huge interest in both social media as well as in traditional publication, these glasses started appearing everywhere. After the grand event passed pairs of these glasses ended up in many a lost shelf or drawer to be forgotten.",
```

Comment:
This description is the story of why you're adding them to the game, not the description of the item itself.  Please just describe the item.